spot-client/src/common/app-state/config/default-config.js

```diff
@@ -62,6 +62,15 @@ export default {
         TRANSMISSION_DELAY: undefined
     },
 
+    UPDATES: {
+        START_HOUR: typeof process.env.UPDATE_START_HOUR === 'undefined'
+            ? 2
+            : process.env.UPDATE_START_HOUR,
+        END_HOUR: typeof process.env.UPDATE_END_HOUR === 'undefined'
+            ? 20
```

Comment:
why `20` by default ? shouldn't it be night hours ? from 2 - 5 or 2 - 4 ?

Reply:
I think I rushed this PR a bit more than usual. The 20 was set for debugging.